### DISCUSSION TITLE
[Enhancement] Apple/Android watch project

### DISCUSSION THREAD

**Discussion by tbonham** [dotnet/maui#80] [General]
## Summary
Please add the option to include Apple or Android watch projects in the default project creation (ie dotnet new maui). 

By having everything one project as you are already doing it would make code sharing even simpler.

**Comment by lukewis** [comment]
I have a new project booting up that will depend heavily on Apple/Android watch capabilities.  Would love to know what the official stance is on Watch (or even other IOT) support.
